src/librustc_codegen_utils/symbol_names.rs

```diff
@@ -436,6 +437,13 @@ impl Printer<'tcx, 'tcx> for SymbolPrinter<'_, 'tcx> {
         Ok(self)
     }
 
+    fn print_const(
+        self,
+        ct: &'tcx ty::Const<'tcx>,
+    ) -> Result<Self::Const, Self::Error> {
+        self.pretty_print_const(ct)
```

Comment:
This seems risky, with the new printing. I think it should just print integer values, or otherwise a `_`.

Reply:
done